gateway-applications/src/main/resources/applications/webshell/app/js/xterm.js

```diff
@@ -0,0 +1,5165 @@
+/**
```

Comment:
This should be minified at minimum - even better if we aren't copying full binary files from elsewhere. 

The link in the source (https://github.com/chjj/term.js) is outdated and the new fork is https://github.com/xtermjs/xterm.js

There doesn't look to be any details as to what version this actually is?

I also think there needs to be an edit to the `LICENSE` file since this is under a different license?